### ISSUE TITLE
Float label on input kendoTextBox with Chrome Autocomplete

### ISSUE THREAD

**Issue by LeoLetourneur** [telerik/kendo-angular#2123]
**Describe the bug**
I have a login form with two field (email, password) (and float labels) like that: 
![Capture1](https://user-images.githubusercontent.com/5477820/54208519-5faa4900-44dc-11e9-9e2d-828e9896ad4e.PNG)

When Chrome auto completes these fields, float labels stay on the bottom of the input:
![Capture3](https://user-images.githubusercontent.com/5477820/54208608-87011600-44dc-11e9-9a2b-d65172f82cc1.PNG)

Maybe change event is not catch by the input... ?

**To Reproduce**
I can't reproduce on stackblitz, autocomplete not working.

**Expected behavior**
Float labels should be on top like that:
![Capture2](https://user-images.githubusercontent.com/5477820/54209560-47d3c480-44de-11e9-8b68-78eaf6108698.PNG)

**Browser**
 - OS: Windows 10
 - Browser Chrome
 - Version 72.0.3626.121

**Additional context**
@progress/kendo-angular-inputs": "4.2.0",

**Comment by danielkaradachki** [comment]
I tried to replicate the problem but to no avail. Can the problem be replicated on your side with this example:

https://stackblitz.com/edit/angular-qwjymy?file=app%2Fapp.component.ts

**Comment by LeoLetourneur** [comment]
Hi, no I can't replicate on Stackblitz. Chrome autocomplete is not working. I can try to make a repo for this.

**Comment by danielkaradachki** [comment]
Strange, the browser autocomplete works in the stackblitz example at least on my side. A runnable sample demonstrating the problem will be appreciated.

**Comment by LeoLetourneur** [comment]
Float labels on inputs is only depending on @progress/kendo-angular-inputs ?

**Comment by danielkaradachki** [comment]
Yes, the [textbox container component](https://www.telerik.com/kendo-angular-ui/components/inputs/api/TextBoxContainerComponent/) which provides this functionality is part of the inputs package.

**Comment by LeoLetourneur** [comment]
Hi,

My repro repo is here : https://github.com/LeoLetourneur/KendoErrors

Some info in readme:
### Float labels with autocomplete password
Using Chrome Version 73.0.3683.86 (Build officiel) (64 bits)
- Start
- Go to http://localhost:4200/login
- Fill any email with password '0123456' => redirect to home
- Chrome will ask you to save credentials => save it
- Go back to http://localhost:4200/login (by refreshing app, not using back button)

**The bug :**   
When page load, float labels stay in initial position (middle of the field) until user click anywhere on the page

**Expected :**   
Float labels autocomplete go to top before user click

Many thanks

**Comment by danielkaradachki** [comment]
Thanks for the example. I was able to replicate the problem. At least Chrome does not seem to trigger any of the usual events when the value is updated and the form value is also out of sync. We will investigate the case further and will update the issue once there is additional information or a fix.

**Comment by elena-gancheva** [comment]
Fixed in `v6.3.0` of the package.
